src/test/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItemSpec.scala

```diff
@@ -48,4 +50,24 @@ class EasyStageFileItemSpec extends FlatSpec with Matchers {
     val args = "src/test/resources/example.csv outdir".split(" ")
     getSettingsRows(new FileItemConf(args)).get.size shouldBe 5
   }
+
```

Comment:
This test really needs to be more specific. Just checking that the entire contents of a file is as a stored example is bound to give false positives (i.e. tests that fail although nothing is broken, just the output changed in some trivial way). The title of the test is also a clear indication that this is not testing something clearly defined.

There are many invariants that can be tested here. To name just a few:
* Of all the files in the list: that they must exist after the test
* that the id of the folder appears where expected in these files
* that the size appears where expected

Better even would be to test lower level functions (`createFileSdo`, `createFolderSdo`). Of course, that doesn't mean that below test cannot be done as well.